.github/workflows/release-and-publish.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       with:
         go-version: 1.18.2
     # Check if the newest tag already exists
-    - uses: mukunku/tag-exists-action@v1.0.0
+    - uses: mukunku/tag-exists-action@v1.1.0
```

Comment:
It's best to use the sha instead of a tag for an action not owned by us

https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions